### PR TITLE
make-tools: fix broken go.mod dep on OpenAPIv2

### PR DESCRIPTION
**Pull request by hunchback**
fix broken deps effecting CI builds

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,48 +3,62 @@ module github.com/ibm/the-mesh-for-data
...
+	k8s.io/kubectl v0.19.0-alpha.3 // indirect
+	k8s.io/utils v0.0.0-20201110183641-67b214c5f920 // indirect
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/cli-utils v0.20.4
 	sigs.k8s.io/controller-runtime v0.6.2
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace helm.sh/helm/v3 v3.2.4 => github.com/the-mesh-for-data/helm/v3 v3.2.4-hunchback2
+replace helm.sh/helm/v3 v3.5.0 => github.com/hunchback/helm/v3 v3.5.0-hunchback
```
Comment (roee88) [inline go.mod:61]: but you have `helm.sh/helm/v3 v3.3.4` in required section. Also did you run `go mod tidy`?

**Review by roee88 — COMMENTED** [review]
I'm getting completely different results after adding `replace helm.sh/helm/v3 v3.5.0 => github.com/hunchback/helm/v3 v3.5.0-hunchback` and changing `helm.sh/helm/v3 v3.2.4` to `helm.sh/helm/v3 v3.5.0`: 

```
module github.com/ibm/the-mesh-for-data

go 1.13

require (
	cloud.google.com/go v0.66.0 // indirect
	emperror.dev/errors v0.7.0
	github.com/IBM/go-sdk-core v1.1.0
	github.com/IBM/satcon-client-go v0.0.0-20210107134702-1be071817792
	github.com/Microsoft/go-winio v0.4.15 // indirect
	github.com/containerd/continuity v0.0.0-20201119173150-04c754faca46 // indirect
	github.com/ghodss/yaml v1.0.0
	github.com/go-chi/chi v4.1.2+incompatible
	github.com/go-chi/render v1.0.1
	github.com/go-logr/logr v0.2.0
	github.com/gogo/protobuf v1.3.2
	github.com/golang/protobuf v1.4.3
	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
	github.com/hashicorp/vault/api v1.0.4
	github.com/machinebox/graphql v0.2.2
	github.com/matryer/is v1.4.0 // indirect
	github.com/onsi/ginkgo v1.14.2
	github.com/onsi/gomega v1.10.3
	github.com/opencontainers/runc v1.0.0-rc9 // indirect
	github.com/pkg/errors v0.9.1
	github.com/robfig/cron v1.2.0
	github.com/stretchr/testify v1.6.1
	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 // indirect
	google.golang.org/grpc v1.31.1
	gopkg.in/yaml.v2 v2.3.0
	helm.sh/helm/v3 v3.5.0
	istio.io/api v0.0.0-20210107192247-a205c627e4b9 // indirect
	istio.io/client-go v0.0.0-20200128004641-c87542c7dc1d
	istio.io/gogo-genproto v0.0.0-20210107204948-697d6f912366 // indirect
	k8s.io/api v0.20.1
	k8s.io/apimachinery v0.20.1
	k8s.io/cli-runtime v0.20.1
	k8s.io/client-go v0.20.1
	rsc.io/letsencrypt v0.0.3 // indirect
	sigs.k8s.io/cli-utils v0.20.4
	sigs.k8s.io/controller-runtime v0.6.2
	sigs.k8s.io/yaml v1.2.0
)

replace helm.sh/helm/v3 v3.5.0 => github.com/hunchback/helm/v3 v3.5.0-hunchback

replace github.com/onsi/gomega => github.com/onsi/gomega v1.10.0

replace github.com/google/addlicense => github.com/the-mesh-for-data/addlicense v0.0.0-20200913135744-636c44b42906

```

File: go.mod (hunk 1 above)
Comment (hunchback) [reply go.mod:61]: I am going to re-run and then run `make verify` to ensure it works

**Comment by hunchback** [comment]
@roee88 build has failed due to a separate (unrelated issue) but the above fix addressed the outstanding go.mod isse - please merge:

```
rror: failed to do request: Head "https://kind-registry:5000/v2/m4d-system/m4d-template/blobs/sha256:0481db428626daeb51155345252d2d279ef2ade4856990066cc3fb3fdf4474d1": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
```

**Review by roee88 — APPROVED** [review]

**Pull request merged**